Assets/StreamingAssets/Quests/A0C00Y00.txt

```diff
@@ -38,9 +39,7 @@ AcceptQuest:  [1002]
 <ce>                       bother trying to find me.
 
 QuestFail:  [1003]
-<ce>             Don't you have a letter to 'deliver' for me?
-                                     <--->
-<ce>         Come back when you have completed your mission, %pcf.
+<ce>             You had a letter to 'deliver' by now.
 
```

Comment:
Is it possible to make it so the original messages here show if you click the quest-giver before completing the mission? Looks like that is what they were intended for.

Reply:
I was actually curious about this one too. This text block with a`<--->` separator should randomly display one of variants of flavour text. Was that not working in tests Jay?

Reply:
Yes, these 1003 comments are often written in first-person. There appears to have been some difficulty among the devs about how to use 1003 and 1009. To begin, Daggerfall's quest endings are instantaneous, so if you fulfill the failure condition there's never an opportunity to hear the failure statement from the questgiver, which is what 1003 appears to intend here. However, the devs never included a "say 1003" at any time, failure or not. 1003 is almost entirely abandoned. I think it's something they asked Ted Peterson to do and then never elaborated on it.

1009 is a separate case, because it tends to use <ce> flags. 1009 would appear when you engage the questgiver in ordinary conversation and would not execute during a quest. <ce> flags are not to be used for ordinary conversation; we use normal unflagged strings for those.

I added a third-person 1003 here because classic often lets time run out on a quest without any forewarning, and then people get confused and say, "I went to the town but the person wasn't there," and we don't know what went wrong until we open the quest parser. Some sort of failuyre information should be displayed, and the classic one was out-of-place.

Reply:
Allofich is saying that some quests (like this one) have messages that fire from the questgiver if you try to talk to them while the quest is unfinished. They are not responses to failure, and if they are fired that way, it is an error in the quest file. As an example of how they should work, see how 1015 fires in a0c00y08. These lines should be retained and fired off appropriately if possible.

1008 and 1009 I think show up in the conversation window when you talk to the person post-quest. They do not fire as click-through messages.

I don't think adding text to quest files is the right way to go about letting the player know when quest time expires. There should probably be a one-size-fits-all message that fires in the scrolling infotext area. Something like "Time has run out on your agreement with _qgiver_". The journal has an area for failed quests to be shown now.

Reply:
I see now. I've looked through the 1003 messages for every classic quest, and universally they're a "get-out-of-here" message the game never implemented. I can do that if that's an improvement.

There are a few occasional instances when classic tells you you've failed a quest due to time-out, but I think that's a big defect. Whether your solution or something else gets added, there really should be some alert to the player about it.